### ISSUE TITLE
Memory leak issue

### ISSUE THREAD

**Issue by Ejz** [SalvatorePreviti/roaring-node#22]
```
const RoaringBitmap = require('roaring/RoaringBitmap32');

for (let i = 0; i < 1E10; i++) {
    RoaringBitmap.andNot(new RoaringBitmap([1, 2, 3]), new RoaringBitmap([1, 2]));
}
```

Simple node script after 5 minutes is killed by system. Memory leak.

**Comment by SalvatorePreviti** [comment]
will have a look, thanks

**Comment by SalvatorePreviti** [comment]
@Ejz from a first analysis I didn't find the memory leak, I am still investigating and trying it out.
Can you specify which version of NodeJS are you using and which OS?

**Comment by SalvatorePreviti** [comment]
`  roaring_bitmap_t * r = roaring_bitmap_andnot(&a->roaring, &b->roaring);
  if (r == nullptr) {
    return v8utils::throwTypeError(isolate, "RoaringBitmap32::andnot failed materalization");
  }

  ra_clear(&self->roaring.high_low_container);
  self->roaring.high_low_container = std::move(r->high_low_container);
  free(r); // this was missing.`

**Comment by SalvatorePreviti** [comment]
Released version 1.0.6 that fix the issue - I am closing the issue because I think is resolved, if you find anything else, please, open another :) thanks.
